components/ChallengeMaterial.tsx

```diff
@@ -51,18 +59,26 @@ export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
   )
 }
 
-export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = props => {
-  const content = (
-    <div className="card-body">
-      <h1 className="challenge-question-card__title">{props.title}</h1>
-      <div>
-        <p className="challenge-question-card__question bg-light p-3 mt-3">
-          {props.question}
-        </p>
+export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = ({
+  currentChallenge
+}) => {
+  return (
+    <div className="card shadow-sm border-0">
+      <div className="card-body challenge-question">
+        <h1 data-testid="challenge-question-title" className="card-title">
+          {currentChallenge!.title}
+        </h1>
+        <div>
+          <p
+            data-testid="challenge-question-description"
+            className="card-question bg-light p-3 mt-3"
+          >
+            {currentChallenge!.description}
```

Comment:
Not sure if I understand the usage of `currentChallenge!` is this the same as forcefully unwrapping an optional when you know there is a value there?

Reply:
updated the types so we good!